sdk/storage/Azure.Storage.Common/src/Shared/Errors.Clients.cs

```diff
@@ -84,5 +85,16 @@ public static void VerifyHttpsTokenAuth(Uri uri)
                 throw new ArgumentException("Cannot use TokenCredential without HTTPS.");
             }
         }
+
+        public static class ClientSideEncryption
+        {
+            public static InvalidOperationException TypeNotSupported(Type type)
```

Comment:
What is the difference between this and EncryptionErrors? Is one more related to client side encryption itself whereas this is more client related?

Reply:
Leftover from split packages. Will cleanup before merge.